### DISCUSSION TITLE
Bed leveling defaults to off on a restart

### DISCUSSION THREAD

**Discussion by hippocratical** [Jyers/Marlin#863] [Q&A] [answered]
Amazing firmware - and the bed mesh levelling works great. It seems to default to being off (checkbox not selected) though when I restart the printer.  

Sometimes I forget to re-enable it and prints go wrong. 

Is there a way to default the bed leveling to always be on after a restart?  

Also/additionally, can bed leveling be enabled through Octoprint?  

EDIT: From further research, I think I have the solution?

In Gcode before print starts I run:
G28
M420 V1
M420 S1

**Comment by Jyers** [comment, marked as answer]
Yup, you got it! you don't need the V1, but M420 S1 will enable the mesh before your print :)
